new-site/src/components/Playground.js

```diff
@@ -0,0 +1,226 @@
+        const textArea = getTextArea(viewPortRef.current);
+        textArea.focus();
+      }
+    },
+    [getTextArea],
+  );
+
+  const onTextAreaBlur = useCallback((e) => {
+    // Set viewport focus on textarea Esc keypress (no related target)
+    if (viewPortRef.current && !e.relatedTarget) {
+      viewPortRef.current.focus();
+    }
+  }, []);
+
+  const copy = () => {
+    if (viewPortRef.current && copyButtonRef.current) {
+      const textArea = getTextArea(viewPortRef.current);
+      textArea.focus();
+      textArea.select();
+
+      try {
+        document.execCommand('copy');
+        if (copyButtonRef.current) {
+          copyButtonRef.current.focus();
+        }
+      } catch (err) {
+        console.warn('Oops, unable to copy');
+      }
+      clearSelection();
+    }
+  };
+
+  const reset = () => {
+    onChange(initialCode);
+    setResetCount(resetCount + 1);
+    if (viewPortRef.current) {
+      viewPortRef.current.focus();
+    }
+  };
+
+  useEffect(() => {
+    if (viewPortRef.current) {
+      const textArea = getTextArea(viewPortRef.current);
+      textArea.setAttribute('aria-describedby', helperTextId);
+      textArea.setAttribute('tabIndex', '-1');
+      textArea.addEventListener('blur', onTextAreaBlur);
+    }
+  }, [getTextArea, helperTextId, onTextAreaBlur]);
+
+  return (
+    <>
+      <div className="playground-block-editor">
+        <div // eslint-disable-line jsx-a11y/no-static-element-interactions
+          className="playground-block-editor-viewport"
+          tabIndex={0} // eslint-disable-line jsx-a11y/no-noninteractive-tabindex
+          onKeyDown={onFocusKeyDown}
+          ref={viewPortRef}
+        >
+          <div className="playground-block-editor-code">
+            <div className="playground-block-editor-texts">
+              <label className="playground-block-editor-label" htmlFor={textAreaId}>
+                Editable code example
+              </label>
+              <span className="playground-block-editor-helper" id={helperTextId}>
+                Press Enter to start editing. Press Esc to stop editing.
+              </span>
+            </div>
+            <div className="playground-block-editor-scrollbox" tabIndex={-1}>
+              <LiveEditor
+                key={resetCount}
+                onChange={onChange}
+                style={{
+                  background: 'none',
+                  padding: '0 0 var(--spacing-s)',
+                  overflow: 'visible',
+                  whiteSpace: 'nowrap',
+                  width: 'max-content',
+                  minWidth: '100%',
+                  fontSize: '16px'
+                }}
+                textareaId={textAreaId}
+                theme={theme}
+              />
+            </div>
+          </div>
+        </div>
+      </div>
+      <div className="playground-block-buttons">
+        <Button ref={copyButtonRef} variant="secondary" size="small" tabIndex={0} onClick={copy}>
```

Comment:
Do we need the `tabIndex={0}` here? Buttons should be focusable by default.

Reply:
- [ ] remove unnecessary tabIndexes